frontend/package.json

```diff
@@ -52,6 +53,7 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^12.1.2",
     "@types/jest": "^27.0.2",
+    "@types/jitsi-meet": "^2.0.2",
```

Comment:
This should be in another PR because it would be nice to take a look at those types. The package name is jitsi-meet and we are not using jitsi-meet. Our javascript library is lib-jitsi-meet